alignment/match_cells.m

```diff
@@ -0,0 +1,65 @@
+function matched_cells = match_cells(sources,m_base_to_others)
+% Compute mutual matching matrix using pairwise cell overlap
+% matrices
+%
+% inputs:
+%   sources: 1 x [number of days] cell array containing paths to the
+%       folders of the days of interest. The base day (the one to which all
+%       other days are matched to) is taken to be the the first folder.
+%   m_base_to_others: 1 x [number of days-1] cell array where each entry has
+%       the pairwise overlap matrix between the base day and another.
+%       m_base_to_others{k} must match with sources{k+1}.
```

Comment:
Just to be clear, this is the "overlap matrix" as returned by [`run_alignment`](https://github.com/inanhkn/analysis/blob/master/alignment/run_alignment.m#L81) as `info.overlap_matrix` (and _not_ `m1to2`)?